CMakeLists.txt

```diff
@@ -29,9 +29,14 @@ target_link_libraries(effil -lpthread ${LUA_LIBRARY} -ldl)
 set_target_properties(effil PROPERTIES SUFFIX .so)
 
 set(GENERAL "-std=c++14 -DSOL_EXCEPTIONS_SAFE_PROPAGATION")
-set(ENABLE_WARNINGS "-Wall -Wextra -pedantic")
+if(NOT MSVC)
+    set(ENABLE_WARNINGS "-Wall -Wextra -pedantic")
+endif()
 set(CMAKE_CXX_FLAGS "${EXTRA_FLAGS} ${CMAKE_CXX_FLAGS} ${GENERAL} ${ENABLE_WARNINGS}")
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Werror -O0 -g -UNDEBUG")
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -g -UNDEBUG")
```

Comment:
As far as I can see Visual Studio ignores these flags:
> cl : Command line warning D9002: ignoring unknown option '-O0' [C:\projects\effil\effil.vcxproj]
> cl : Command line warning D9002: ignoring unknown option '-g' [C:\projects\effil\effil.vcxproj]

So we can also put them under `If` and we can make single `if` instead of two